### PR TITLE
[14.0][MIG] analytic_cost_revenue

### PR DESCRIPTION
**Pull request by DhavalTforgeFlow**

### CODE REVIEW

File: analytic_cost_revenue/models/analytic.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline analytic_cost_revenue/models/analytic.py]:
```suggestion
# Copyright 2017-23 ForgeFlow S.L.
```

**Review by AaronHForgeFlow — APPROVED** [review]
Functional test. Just a minor thing.

**Review by JuanyDForgeflow — APPROVED** [review]
Code + Functional Review

File: analytic_cost_revenue/models/analytic.py (hunk 1 above)
Comment (JuanyDForgeflow) [reply analytic_cost_revenue/models/analytic.py]: @AaronHForgeFlow This is changed now

**Pull request merged**